### PR TITLE
Fix dependencies in integration tests

### PR DESCRIPTION
**Pull request by atburke**
This PR moves some type/function definitions in integration tests to fix compilation.

### CODE REVIEW

File: integration/helpers.go [hunk 1]
```diff
@@ -77,6 +78,218 @@ const (
...
+	me *user.User
+	// priv/pub pair to avoid re-generating it
+	priv []byte
+	pub  []byte
+	// log defines the test-specific logger
+	log utils.Logger
+}
+
+func newSuite(t *testing.T) *integrationTestSuite {
+	SetTestTimeouts(time.Millisecond * time.Duration(100))
```
Comment (zmb3) [inline integration/helpers.go:96]: I think this can be removed. @jakule ?

File: integration/helpers.go [hunk 2]
```diff
@@ -77,6 +78,218 @@ const (
 	Host     = "localhost"
 )
 
+const (
+	HostID = "00000000-0000-0000-0000-000000000000"
+	Site   = "local-site"
+)
+
+type integrationTestSuite struct {
```
Comment (zmb3) [inline integration/helpers.go:86]: This isn't used for all tests, is it?

If not, can we give it a more specific name?

File: integration/helpers.go (hunk 1 above)
Comment (Joerger) [reply integration/helpers.go:96]: It looks like this was a merge conflict, it's not in `master`.

**Review by Joerger — COMMENTED** [review]
This seems a bit backwards to me, shouldn't `integrationTestSuite` remain in `integration_test.go`? It looks like `createTrustedClusterPair` is the only thing that needs to be changed/moved, sorry if I'm missing something else.

Can we instead move `createTrustedClusterPair` to `integration_test.go` or change the function signature to not require `integrationTestSuite`?

File: integration/helpers.go (hunk 1 above)
Comment (tcsc) [reply integration/helpers.go]: `SetTestTimeouts()` call should definitely be here. If it's not in `master` then it probably explains some of the integration test timeouts we've been seeing.

Comment (Joerger) [reply integration/helpers.go]: Here's the PR for it from today, though I haven't looked into the details - https://github.com/gravitational/teleport/pull/13312

Comment (jakule) [reply integration/helpers.go]: I removed it today as Brian mentioned.

**Review by Joerger — APPROVED** [review]

**Review by rosstimothy — APPROVED** [review]

**Pull request merged**